frontend/lib/justfix-site.tsx

```diff
@@ -120,8 +120,11 @@ const JustfixRoute: React.FC<RouteComponentProps> = (props) => {
         exact
         component={LogoutPage}
       />
-      {toPLRoute(getOnboardingRouteForIntent(OnboardingInfoSignupIntent.LOC))}
-      <PLRoute
+      {toPLRoute(getOnboardingRouteForIntent(OnboardingInfoSignupIntent.LOC), {
+        locales: ["en"],
+        wipLocales: ["es"],
+      })}
+      <Route
```

Comment:
Hmm I'm noticing the last line (line 127) is now a `<Route>` instead of a `<PLRoute>`, is that intentional?  It seems like it might make the `/loc/` routes suddenly internationalized rather than redirecting to English.

Reply:
Reverting this!